app/build.gradle

```diff
@@ -11,9 +11,10 @@ ext {
 
     supportVersion = '26.1.0'
     constraintLayoutVersion = '1.0.2'
-    archComponentVersion = '1.0.0-rc1'
+    archComponentVersion = '1.0.0'
     rxJavaVersion = '2.1.6'
     rxAndroidVersion = '2.0.1'
+    reactiveAndroidVersion = '1.0.0-alpha3'
```

Comment:
wonder why is it `alpha` still, what do you have in mind that is still lacking from this library❓

Reply:
It is lacking tests! Also, some of the APIs are not fully migrated yet.